Blockzilla/UIComponents/URLBar.swift

```diff
@@ -426,10 +426,19 @@ class URLBar: UIView {
         }
     }
 
+    private func identifyKeyboardNameTelemetry() {
+        enum Keyboard: String { case `default` = "Default Keyboard:", custom = "Custom Keyboard:" }
+        guard UIApplication.textInputMode?.responds(to: NSSelectorFromString("identifier")) == true else { return }
+        guard var identifier = UIApplication.textInputMode?.perform(NSSelectorFromString("identifier")).takeRetainedValue() as? String else { return }
+        identifier = identifier.contains("@sw=") ? Keyboard.default.rawValue + identifier : Keyboard.custom.rawValue + identifier
```

Comment:
We can probably put this guard logic on a method in a `UIApplication` extension which return a keyboard identifier.